package.json

```diff
@@ -0,0 +1,85 @@
+    },
+    "dependencies": {
+        "quill-blot-formatter": "1.0.5"
+    },
+    "peerDependencies": {
+        "quill": "1.3.7"
+    },
+    "description": "Common patterns, functions, etc... used when building react applications",
+    "devDependencies": {
+        "@testing-library/jest-dom": "5.5.0",
+        "@testing-library/react": "10.0.4",
+        "@types/faker": "4.1.12",
+        "@types/jest": "25.1.5",
+        "@types/node": "13.11.0",
+        "@types/quill": "2.0.4",
+        "@types/rosie": "0.0.37",
+        "andculturecode-javascript-core": "0.3.6",
+        "andculturecode-javascript-testing": "0.0.1",
+        "cross-env": "6.0.3",
+        "cypress": "3.8.3",
+        "faker": "4.1.0",
+        "jest": "25.5.4",
+        "jest-extended": "0.11.5",
+        "prettier": "1.19.1",
+        "rimraf": "2.6.2",
+        "rosie": "2.0.1",
+        "ts-jest": "25.5.1",
+        "ts-loader": "8.0.4",
+        "tslint": "6.1.2",
+        "tslint-config-prettier": "1.18.0",
+        "typedoc": "0.17.6",
+        "typedoc-plugin-markdown": "2.2.17",
+        "typescript": "3.8.3",
+        "webpack": "4.44.2",
+        "webpack-cli": "3.3.12"
+    },
+    "files": [
+        "dist",
+        "dist-global"
+    ],
+    "homepage": "https://github.com/AndcultureCode/AndcultureCode.JavaScript.Quill#readme",
+    "jest": {
+        "preset": "ts-jest"
+    },
+    "keywords": [
+        "JavaScript",
+        "Node",
+        "React"
+    ],
+    "license": "ISC",
+    "main": "dist/index",
+    "name": "andculturecode-javascript-quill",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/AndcultureCode/AndcultureCode.JavaScript.Quill.git"
+    },
+    "scripts": {
+        "build": "tsc --pretty && webpack && npm run validate-peer-dependencies",
+        "clean": "rimraf dist && rimraf coverage",
+        "configure:git": "echo Ensuring git hooksPath is set to .githooks directory && git config core.hooksPath .githooks && chmod +x .githooks/*",
+        "coverage": "jest ./src --coverage",
+        "docs": "typedoc --mode file --excludeExternals --plugin typedoc-plugin-markdown --readme none",
+        "format": "prettier --write \"src/**/*.ts\" --trailing-comma es5",
+        "postbuild": "npm run docs",
+        "postpublish": "cross-env-shell \"git add -A && git commit -m \"$npm_package_version\" && git push origin main\"",
+        "prebuild": "npm run clean && npm run format && echo Using TypeScript && tsc --version",
+        "prepublishOnly": "npm run build",
+        "test": "jest ./src",
+        "test-cypress": "cypress run",
+        "validate-peer-dependencies": "node validate-peer-dependencies.js",
+        "watch": "npm run build -- --watch",
+        "watch:coverage": "jest ./src --coverage --watch",
+        "watch:test": "jest ./src --watch"
+    },
+    "types": "dist/index",
+    "version": "0.3.5"

```

Comment:
Probably best to start this back at 0.0.1 before it starts being published out 😛

Reply:
oops, my bad 👍